### PR TITLE
Parse pubDate without time

### PR DESCRIPTION
**Pull request by stchris**
Another weirdness example taken from https://rust.careers/rss

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -91,6 +91,11 @@ func parseTime(raw string) (t time.Time, err error) {
 		return t, nil
 	}
 
+	t, err = time.Parse("2006-01-02", raw)
+	if err == nil {
```
Comment (fgeller) [inline main.go:95]: and not like me, you got the `==` right 😉  👍

**Comment by fgeller** [comment]
Awesome @stchris , thanks a lot! I wonder how long it'll take to make it to more than 10 different time layouts 😅 

Are you interested in commit rights? Wouldn't be bound to any expectations from my side.

**Comment by stchris** [comment]
> Awesome @stchris , thanks a lot! I wonder how long it'll take to make it to more than 10 different time layouts 😅 

Let's see 😬

> Are you interested in commit rights? Wouldn't be bound to any expectations from my side.

Nah, I'm good. PRs make sense anyway, let's see where this goes.

File: main.go (hunk 1 above)
Comment (stchris) [reply main.go:95]: By copying and pasting ... 😆

**Comment by fgeller** [comment]
sweet, just bring it up if something changes 👍

**Pull request merged**